CLAMSSpeciesFix.py

```diff
@@ -107,7 +111,7 @@ def __init__(self, parent=None):
         #  from the window's init method.
         initTimer = QTimer(self)
         initTimer.setSingleShot(True)
-        self.connect(initTimer, SIGNAL("timeout()"), self.formInit)
+        self.initTimer.clicked.connect(self.formInit)
```

Comment:
Same issue here with the incorrect signal name. This is a timer and the signal we're connecting is `timeout`